core/src/main/java/org/testcontainers/containers/GenericContainer.java

```diff
@@ -217,7 +187,7 @@ public class GenericContainer<SELF extends GenericContainer<SELF>>
      * The approach to determine if the container is ready.
      */
     @NonNull
-    protected org.testcontainers.containers.wait.strategy.WaitStrategy waitStrategy = Wait.defaultWaitStrategy();
+    protected WaitStrategy waitStrategy = Wait.defaultWaitStrategy();
```

Comment:
Let's add a comment that we have to keep this field in `GenericContainer` for now to avoid a breaking change.